ironfish/src/network/peers/peerCandidates.ts

```diff
@@ -76,8 +76,8 @@ export class PeerCandidates {
     }
   }
 
-  shufflePeerCandidates(): string[] {
-    return ArrayUtils.shuffle([...this.map.keys()])
+  shufflePeerCandidates(): [Identity, PeerCandidate][] {
```

Comment:
I don't think this is a good change - previously we were allocating an array of size this.map.size * identity length, now we're allocating an array of this.map.size * (identity length + peer candidate size) which is substantially more, just to avoid looking the candidate up from a hashmap we already have. Readability is good, but not at the cost of performance

Reply:
That's fair, I changed it back. Now that specific change just has a few variable name changes